src/propslib/RegistrySerialization.cpp

```diff
@@ -7,7 +7,7 @@
 
 #pragma hdrstop
 
-#define SET_FIELD_AND_SIZE(x) FIELD_OFFSET(Settings, x), RTL_FIELD_SIZE(Settings, x)
+#define SET_FIELD_AND_SIZE(x) UFIELD_OFFSET(Settings, x), RTL_FIELD_SIZE(Settings, x)

```

Comment:
I'm not sure why this wasn't a problem before, but `FIELD_OFFSET` returns a `LONG`, which then has to be downcast to a `DWORD`, which the audit doesn't like. `UFIELD_OFFSET` returns a `DWORD`.